tests/integration/test_core.py

```diff
@@ -353,3 +373,69 @@ async def test_openstack_object_storage_plugin(
             assert (
                 requests.get(url).content == image
             ), "image downloaded from WordPress should match the image uploaded"
+
+
+@pytest.mark.asyncio
+async def test_openstack_akismet_plugin(
+    ops_test: pytest_operator.plugin.OpsTest,
+    application_name,
+    default_admin_password,
+    unit_ip_list,
+    akismet_api_key,
+):
+    """
+    arrange: after WordPress charm has been deployed, db relation established
+    act: update charm configuration for Akismet plugin
+    assert: Akismet plugin should be activated and spam detection function should be working
+    """
+    application = ops_test.model.applications[application_name]
+    await application.set_config({"wp_plugin_akismet_key": akismet_api_key})
+    await ops_test.model.wait_for_idle()
+
+    for unit_ip in unit_ip_list:
+        wp = WordpressClient(
+            host=unit_ip, username="admin", password=default_admin_password, is_admin=True
+        )
+        post = wp.create_post(secrets.token_hex(8), secrets.token_hex(8))
+        wp.create_comment(
+            post_id=post["id"], post_link=post["link"], content="akismet-guaranteed-spam"
+        )
+        wp.create_comment(post_id=post["id"], post_link=post["link"], content="test comment")
+        assert (
+            len(wp.list_comments(status="spam", post_id=post["id"])) == 1
+        ), "Akismet plugin should move the triggered spam comment to the spam section"
+        assert (
+            len(wp.list_comments(post_id=post["id"])) == 1
+        ), "Akismet plugin should keep the normal comment"
+
+
+@pytest.mark.asyncio
+async def test_openstack_openid_plugin(
+    ops_test: pytest_operator.plugin.OpsTest,
+    application_name,
+    default_admin_password,
+    unit_ip_list,
+    openid_username,
+    openid_password,
+):
+    """
+    arrange: after WordPress charm has been deployed, db relation established
+    act: update charm configuration for Akismet plugin
+    assert: An Ubuntu One OpenID account should be able to associate with a WordPress user
+    """
+    application = ops_test.model.applications[application_name]
+    await application.set_config({"wp_plugin_openid_team_map": "site-sysadmins=administrator"})
```

Comment:
Similar to the above, do you think this configuration should be reverted after the test completes? If so, do you think it should be done in a fixture to guarantee that it is reverted?